roles/network_plugin/kube-ovn/defaults/main.yml

```diff
@@ -37,10 +38,16 @@ kube_ovn_central_ips: |-
   {%- endfor %}
 
 kube_ovn_ic_enable: false
-kube_ovn_ic_autoroute: true
-kube_ovn_ic_dbhost: "127.0.0.1"
+kube_ovn_ic_auto_route: true
+kube_ovn_ic_db_host: "127.0.0.1"
```

Comment:
Changing variable names is a breaking change => we should probably leave those alone

Reply:
<img width="495" alt="image" src="https://github.com/user-attachments/assets/53591455-6599-4185-ba64-164e19bcce71" />
j2 will render the value into a file. It will not break anything. it is a key value in Ansible, but, nothing changes in the k8s yaml. right?

Reply:
That's not it.

Files in defaults can be overridden by users in their inventory (that's their point). So if we change the variable name and use the new one, someone who had defined a value different from the default would silently reset to the default on upgrading.

It's ok to have breaking changes when there is a sufficiently :tm: benefit (or we just merged and it was not in a released version). But changing spacing is below the bar of usefulness for making a breaking change, IMO.

Reply:
ok,will roll back